test/navigation.spec.ts

```diff
@@ -155,7 +155,7 @@ describe('navigation', function () {
         ? 'net::ERR_CERT_INVALID'
         : 'net::ERR_CERT_AUTHORITY_INVALID';
 
-    itFailsFirefox('should fail when navigating to bad SSL', async () => {
+    it('should fail when navigating to bad SSL', async () => {
```

Comment:
Oh, I think I incorrectly marked it as `it` instead of 'itFailsFirefox' after I removed `.only` for the repro.